.circleci/config.yml

```diff
@@ -229,20 +229,20 @@ jobs:
           steps:
             - run: ./diff.sh creator-node || (echo "no diff" && circleci-agent step halt)
       # Download and cache dependencies
-      - restore_cache:
-          keys:
-          - v1-dependencies-{{ checksum "creator-node/package.json" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
+      # - restore_cache:
```

Comment:
For some reason CircleCI doesn't like using the old cache with this version of npm. We're getting an error restoring an old cache. For the time being, cache restore and save is disabled, tests are passing